tests/fixtures/auth_keys.py

```diff
@@ -22,7 +22,7 @@
 def print_auth_key_env() -> None:
     """Print environment setting for the auth key."""
     auth_key = generate_jwk().export(private_key=False)
-    print(f"ARS_AUTH_KEY={auth_key}")
+    print(f"ARS_AUTH_KEY={auth_key!r}")
```

Comment:
```suggestion
    ARS_AUTH_KEY = generate_jwk().export(private_key=False)
    print(f"{ARS_AUTH_KEY=!r}")
```
I think you might be able to even omit the !r at that point, but I'm not 100% sure.

Reply:
Good idea, that's one of the features that were newly introduced in Python 3.8.

The formatting with "!r" adds single quotes which are needed so that the JSON does not get mangled when executed as a shell command.